plugins-list.yaml

```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
+# plugins/catalog-backend-module-bitbucket-cloud  ==> wrapper
+# plugins/catalog-backend-module-bitbucket-server  ==> wrapper
 plugins/catalog-backend-module-gcp:
 plugins/catalog-backend-module-gerrit:
-plugins/catalog-backend-module-github-org:
-plugins/catalog-backend-module-github:
-plugins/catalog-backend-module-gitlab:
+# plugins/catalog-backend-module-github  ==> wrapper
+# plugins/catalog-backend-module-github-org  ==> wrapper
+# plugins/catalog-backend-module-gitlab  ==> wrapper
+# plugins/catalog-backend-module-gitlab-org  ==> wrapper
 plugins/catalog-backend-module-incremental-ingestion:
-#plugins/catalog-backend-module-ldap: ==> New backend system support missing
-plugins/catalog-backend-module-msgraph:
+# plugins/catalog-backend-module-ldap  ==> wrapper
+# plugins/catalog-backend-module-logs  ==> wrapper
+# plugins/catalog-backend-module-msgraph  ==> wrapper
 plugins/catalog-backend-module-openapi:
 plugins/catalog-backend-module-puppetdb:
 plugins/catalog-backend-module-scaffolder-entity-model:
 plugins/catalog-backend-module-unprocessed:
-#plugins/catalog-graph: ==> Added as a static plugin in RHDH 
-#plugins/catalog-import: ==> Added as a static plugin in RHDH
+plugins/catalog-common:
+# plugins/catalog-graph ==> Added as a static plugin in RHDH :
+# plugins/catalog-import ==> Added as a static plugin in RHDH
+plugins/catalog-node:
+plugins/catalog-react:
 plugins/catalog-unprocessed-entities:
+plugins/catalog-unprocessed-entities-common:
 plugins/config-schema:
 plugins/devtools:
 plugins/devtools-backend:
+plugins/devtools-common:
 plugins/events-backend:
 plugins/events-backend-module-aws-sqs:
 plugins/events-backend-module-azure:
 plugins/events-backend-module-bitbucket-cloud:
 plugins/events-backend-module-gerrit:
-#plugins/events-backend-module-github: # ==> the new backend system is not exported as default export (Error: Backend plugin is not valid for dynamic loading: it should either export a BackendFeature or BackendFeatureFactory as default export, or export a const dynamicPluginInstaller: BackendDynamicPluginInstaller field as dynamic loading entrypoint)
-#plugins/events-backend-module-gitlab: # ==> the new backend system is not exported as default export (Error: Backend plugin is not valid for dynamic loading: it should either export a BackendFeature or BackendFeatureFactory as default export, or export a const dynamicPluginInstaller: BackendDynamicPluginInstaller field as dynamic loading entrypoint)
+# plugins/events-backend-module-github ==> Added as a static plugin in RHDH
+# plugins/events-backend-module-gitlab ==> Added as a static plugin in RHDH
+plugins/events-backend-test-utils:
+plugins/events-node:
 plugins/example-todo-list:
 plugins/example-todo-list-backend:
+plugins/example-todo-list-common:
 plugins/home:
-plugins/kubernetes:
-plugins/kubernetes-backend:
+plugins/home-react:
+# plugins/kubernetes  ==> wrapper
+# plugins/kubernetes-backend  ==> wrapper
+plugins/kubernetes-cluster:
```

Comment:
Not a plugin, afaict

Reply:
`plugins/kubernetes-cluster` has the role `frontend-plugin`